frontend/site-registry/src/app/constants/Constant.ts

```diff
@@ -1,6 +1,6 @@
-export const FOOTER_TEXT_1 =
-  'The B.C. Public Service acknowledges the territories of First Nations around B.C. and is grateful to carry out our work on these lands. We acknowledge the rights, interests, priorities, and concerns of all Indigenous Peoples - First Nations, Métis, and Inuit - respecting and acknowledging their distinct cultures, histories, rights, laws, and governments.';
-export const FOOTER_TEXT_2 =
-  'We can help in over 120 languages and through Telephone Device For The Deaf (TDD). Call, email or text us, or find a service centre';
-export const INFO_TEXT_1 =
-  'Ea similique eum id illo et commodi. Itaque ducimus accusantium qui consequatur. Vitae nulla impedit laborum suscipit dolor ex at numquam. Optio rerum et enim impedit. Labore deserunt cupiditate deleniti.';
```

Comment:
I tried running this branch locally and I'm getting a bunch of errors about these export missing:

```
ERROR in src/app/components/navigation/Footer.tsx:4:10
TS2305: Module '"../../constants/Constant"' has no exported member 'FOOTER_TEXT_1'.
    2 | import './Footer.css';
    3 | import logo from '../../../app/images/logos/logo-white.png';
  > 4 | import { FOOTER_TEXT_1, FOOTER_TEXT_2 } from '../../constants/Constant';
      |          ^^^^^^^^^^^^^
    5 |
    6 | const Footer = () => {
    7 |   return (
```

Reply:
Let me look into this as I removed few but let me cross check again.

Reply:
I have added them in another commit. Pls feel to check again and let me know in case of any issues.